docs/styles/Vocab/ANSYS/accept.txt

```diff
@@ -32,6 +36,7 @@ Reusability
 Rz
 scopings
 serializer
+Server

```

Comment:
@PProfizi You can try using DPF Server instead of server, otherwise it will complain in all server instances

Reply:
Thx @Revathyvenugopal162! I just saw your comment, and indeed I ended-up doing just that! This is relly useful.
I had to set it as `DPF [Ss]erver` though as otherwise it would not accept `DPF server` anymore.